### PR TITLE
(Feature/Styling) Opportunity Card Updated

### LINKED ISSUE #194: Redesign of the job/opportunity card and make it complement the whole website.
### **Feature request**:

A clear and concise description of the feature.
  The card component in the website which is shown for different jobs and opporutunities doesn't look ideal and the image can be rounded instead of a square which doesn't look good. The details and apply button too can see improvement and look more inclusive and professional.

### **Is your feature request related to a problem? Please describe. (optional)**:
N/A
A clear and concise description of what the problem is.
N/A

### **Describe the solution you'd like**:
I want to make the card much more professional by aligning it so that it looks good and complements the design of this website.

### **Describe alternatives you've considered**:
Will check different alignements and design which look the best and then come to the best design about the card.

### **Describe how this feature will be useful to our readers**:
A clear and concise description of how this feature will be useful for our other readers.
The square image doesn't look more professional and the card could see visual improvements so that users can easily see what is the job, who it is from and how to apply while looking genuinely impressive.

### **Additional context:**

The job card doesn't look professional.
Screenshot from 2021-04-07 12-50-47

<hr/>

**Note:**

- If you want to work on an issue, you should check if it has already been assigned to anyone.

### PR DESCRIPTION
**Pull request by mohitms007**
# Description
Redesign of the job/opportunity card and make it complement the whole website.  Make the card more intutive and easy to see.

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.
I have only made changes to files needed for it that were opporutunity card and its styles. Rest are as it is. I have made the card more visible, fixed the image layout, added some classes and button props and make it visually appealing.

Fixed #194 

## Type of change
Visual Change/ Layout / UI design
Please check options that are relevant to your PR.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

# Checklist:

- [x] I have pulled the latest changes in the repository
- [x] I have squashed my commits
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by DeepanshiD** [comment]
@mohitms007 Looks nice to me can you keep Apply button on the right bottom of card?
Increase shadow a bit on hover, not much just a little.
Remove top blue bar

CC: @abdus

**Comment by mohitms007** [comment]
> @mohitms007 Looks nice to me can you keep Apply button on the right bottom of card?
> Increase shadow a bit on hover, not much just a little.
> Remove top blue bar
> 
> CC: @abdus 

Sure! Will fix it immediately.

**Comment by mohitms007** [comment]
> @mohitms007 Looks nice to me can you keep Apply button on the right bottom of card?
> Increase shadow a bit on hover, not much just a little.
> Remove top blue bar
> 
> CC: @abdus

Fixed it.

File: src/Components/ViewOpportunity/Opportunity.module.css [hunk 1]
```diff
@@ -16,8 +16,6 @@
   /* margin-left: 50%; */
```
Comment (vaishali614) [inline src/Components/ViewOpportunity/Opportunity.module.css:16]: Increase card width to 60rem and decrease margin-left to 5rem.

File: src/Components/ViewOpportunity/Opportunity.module.css [hunk 2]
```diff
@@ -34,12 +32,11 @@ a:hover {
 }
 
 .CardImage {
```
Comment (vaishali614) [inline src/Components/ViewOpportunity/Opportunity.module.css:38]: Align the company logo with the heading on its right.

File: src/Components/ViewOpportunity/Opportunity.module.css [hunk 3]
```diff
@@ -59,6 +56,7 @@ a:hover {
   display: inline-block;
   margin-top: 2rem;
   color: #008dc8;
+  margin-left: 10px;
   font-size: 20px;
```
Comment (vaishali614) [inline src/Components/ViewOpportunity/Opportunity.module.css]: Change font-size to 1.5rem

File: src/Components/ViewOpportunity/Opportunity.module.css [hunk 4]
```diff
@@ -68,17 +66,22 @@ a:hover {
   -moz-hyphen: none;
 }
 
-.Description {
-  margin-top: 25px;
-  margin-left: 35px;
-  hyphens: none;
-  line-break: auto;
+.Subtitle{
+  margin-left: 10px;
```
Comment (vaishali614) [inline src/Components/ViewOpportunity/Opportunity.module.css]: Give it a font-size of 1.2rem

File: src/Components/ViewOpportunity/Opportunity.module.css [hunk 5]
```diff
@@ -123,29 +126,23 @@ a:hover {
...
+  display:flex;
+  justify-content:center;
+  align-items:center;
+  margin-right: 5px;
   float: right;
-  margin-top: 20px;
-  padding: 5px 30px 5px 30px;
-  color: #fff;
-  font-size: 20px;
+  font-size: 15px;
```
Comment (vaishali614) [inline src/Components/ViewOpportunity/Opportunity.module.css]: 1. Change font-size to 1.2rem
2. Add padding: 5px 35px;
3. Add border: 1px solid #008CD8

**Review by vaishali614 — CHANGES_REQUESTED** [review]

File: src/Components/ViewOpportunity/Opportunity.module.css (hunk 2 above)
Comment (vaishali614) [reply src/Components/ViewOpportunity/Opportunity.module.css:38]: Logo still doesn't seem aligned with heading. Also, please check everything for all screen sizes.

Comment (mohitms007) [reply src/Components/ViewOpportunity/Opportunity.module.css:38]: The subtitle of the heading is causing issues with the alignment so I placed the subtitle to the right and aligned items in the center. Also tried to fix card issues and responsiveness of them by changing widths and font size of certain titles.

Comment (vaishali614) [reply src/Components/ViewOpportunity/Opportunity.module.css:38]: I don't think that should be a problem. You can do it this way - 
1. One row: Two columns
2. First column: Image
3. Second column: Title and Subtitle

This should automatically align them

Comment (mohitms007) [reply src/Components/ViewOpportunity/Opportunity.module.css:38]: I did the same in my first commit but still was not aligned properly probably due to some extra margins or padding. Is the company name(subtitle)  on right not suitable?

Comment (vaishali614) [reply src/Components/ViewOpportunity/Opportunity.module.css:38]: Yeah it doesn't look good on right.
Try removing the extra margin/padding. If you're still unable to do so, undo the last changes i.e. subtitle on right and push the other changes.

Comment (mohitms007) [reply src/Components/ViewOpportunity/Opportunity.module.css:38]: Alright! Will Try.

Comment (mohitms007) [reply src/Components/ViewOpportunity/Opportunity.module.css:38]: I have pushed my changes. Placed the subtitle where it was and tried to cancel all margins and align them properly. Also checked up on the mobile versions aswell.

Comment (vaishali614) [reply src/Components/ViewOpportunity/Opportunity.module.css:38]: okay

**Comment by vaishali614** [comment]
@mohitms007 Cards are not visible completely in small screens. Please check this thing. Rest seems good.

4Capture

**Comment by mohitms007** [comment]
> @mohitms007 Cards are not visible completely in small screens. Please check this thing. Rest seems good.
> 
> 4Capture

Fixed it.

Screenshot from 2021-04-13 19-24-51

**Comment by vaishali614** [comment]
@mohitms007 That's the problem with some other screen sizes too. Please check for all screens before you update your PR.
Also, do take latest pull from upstream. Rest everything seems fine. Please update these changes so that I can merge your PR.

**Comment by mohitms007** [comment]
> @mohitms007 That's the problem with some other screen sizes too. Please check for all screens before you update your PR.
> Also, do take latest pull from upstream. Rest everything seems fine. Please update these changes so that I can merge your PR.

Checked all the screens works fine now and pulled the upstream too.

**Review by vaishali614 — APPROVED** [review]

**Pull request merged**